docs/caching/includes/garnet-app-host.md

```diff
@@ -0,0 +1,56 @@
+<!--
```

Comment:
Should we remove this commented out stuff and just publish the minimal version?

Reply:
Oh .. its an include ... never mind.

Reply:
I deleted them anyway, they were just comments for us, wouldn't render as HTML.